python_terraform/terraform.py

```diff
@@ -151,11 +153,15 @@ def destroy(
         force/no-color option is flagged by default
         :return: ret_code, stdout, stderr
         """
+        global_opts = self._generate_default_general_options(dir_or_plan)
         default = kwargs.copy()
-        default["force"] = force
+        # force is no longer a flag in version >= 1.0
+        if self.terraform_version < 1.0:
```

Comment:
Here we use the terraform_version in order to distinguishe between 0.x & 1.x

Reply:
I see 👍